parts/linux/cloud-init/artifacts/components.json

```diff
@@ -369,7 +369,7 @@
       ],
       "windowsVersions": [
         {
-          "renovateTag": "<DO_NOT_UPDATE>",
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/azure/secrets-store/provider-azure",
           "latestVersion": "v1.5.3",
```

Comment:
Those versions are inversed.  

Also I think we need to go back to the drawing board. We are duplicating too many entries for windows and Linux now. I don't think we should copy paste all those settings "renovateTag, previousTag, latest tag".  Those should be identical between Linux and windows (use the same properties. Instead of duplicating )

Reply:
We have the capability to use the same version blob for Windows and Linux, and do this in some places. I plan to go through components.json and update when they are the same. Because windows containers are so much larger than linux containers there are many places where we don't want them to be the same - we want fewer on windows. There are also cases where particular versions (of cni/cns containers in particular) don't work on Windows, so we use an older version.